graylog2-web-interface/src/pages/ShowContentPackPage.jsx

```diff
@@ -95,9 +100,6 @@ const ShowContentPackPage = createReactClass({
               <LinkContainer to={Routes.SYSTEM.CONTENTPACKS.LIST}>
                 <Button bsStyle="info">Content Packs</Button>
               </LinkContainer>
-              <LinkContainer to={Routes.SYSTEM.CONTENTPACKS.edit(encodeURIComponent(lastPack.id), encodeURIComponent(lastPack.rev))}>
```

Comment:
I think something went wrong during the merge conflict, `lastPack` is not used any more, so we can remove the variable.
We should also remove `lastVersion`, as it is only used to get `lastPack` and the function it references is removed in this change.